tests/integration_tests/performance/test_vsock_throughput.py

```diff
@@ -0,0 +1,381 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests the VSOCK throughput of Firecracker uVMs."""
+
+
+import os
+import json
+import logging
+import time
+import concurrent.futures
+import pytest
+from conftest import _test_images_s3_bucket
+from framework.artifacts import ArtifactCollection, ArtifactSet
+from framework.matrix import TestMatrix, TestContext
+from framework.builder import MicrovmBuilder
+from framework.statistics import core, consumer, producer, criteria,  types, \
+    function
+from framework.utils import CpuMap, CmdBuilder, run_cmd, eager_map, \
+    get_cpu_percent, get_cpu_model_name
+import host_tools.network as net_tools
+from integration_tests.performance.vsock_throughput_test_config import \
```

Comment:
Why don't import all?

Reply:
We could, but the python style test keeps getting in the way.
See this comment for a more detailed explanation: https://github.com/firecracker-microvm/firecracker/pull/2338#discussion_r542511361